api/transport/handler.go

```diff
@@ -49,6 +51,12 @@ type HandlerSpec struct {
 	onewayHandler OnewayHandler
 }
 
+// MarshalLogObject implements zap.ObjectMarshaler.
+func (h HandlerSpec) MarshalLogObject(enc zapcore.ObjectEncoder) error {
+	enc.AddString("type", h.t.String())
+	return nil
```

Comment:
What's the motivation for this func returning `error`; looks like you're returning `nil` every time here.